src/api/app/views/layouts/webui/webui2.html.haml

```diff
@@ -0,0 +1,65 @@
+!!!
+%html
+  %head
+    %meta{:content => "text/html; charset=utf-8", "http-equiv" => "Content-Type"}/
+    - if @metarobots
+      %meta{:content => "#{@metarobots}", :name => "robots"}/
+    = favicon_link_tag
+    %title
+      = "#{@pagetitle} - " if @pagetitle
+      = @configuration ? @configuration['title'] : 'Open Build Service'
+    = stylesheet_link_tag 'webui2/webui2'
+    - if content_for?(:head_style)
+      :css
+        #{yield :head_style}
+    = yield :content_for_head
+    = javascript_include_tag 'webui2/application'
+    = javascript_tag do
+      = yield :head_javascript
+      var _paq = _paq || [];
+      $(function() {
+      \#{yield :ready_function}
+      - if @configuration['theme'] == 'bento'
```

Comment:
do you still call the bootstrap theme bento then? :)

Reply:
We plan to keep the visible theme for now and just exchange the underlying technology. Afterwards we can slowly change the Appearance of OBS.
If we change everything at once users would be lost.

Reply:
I understand that part - you can still give it a different name to avoid total confusion in daily converstations :)

Reply:
call it `bs-bento` if you want 🥇

Reply:
Well, that's part of a config that will be a thing for openSUSE, so I guess we need to call it after current openSUSE theme based on bootstrap (as I expect that will be used for reference server ;) )